Resources/doc/resource_owners/fiware.md

```diff
@@ -0,0 +1,35 @@
+Step 2x: Setup FI-WARE
+=========================
+First you will have to register your application on your FI-WARE Identity Management - KeyRock. Check out the
```

Comment:
@ck-1 you could also send a separate PR fixing all existing documentation for such mistakes